### ISSUE TITLE
Need for ShadyCSS

### ISSUE THREAD

**Issue by LarsDenBakker** [Polymer/lit-element#641]
`LitElement` is no longer supporting `@apply` for styles defined using the static `styles` property, but it's still using `shady-render` and calling ` window.ShadyCSS.styleElement(this);` in the `connectedCallback`. 

Why is that still needed? If it's only for supporting `@apply` in regular `<style>` tags I think this could be made configurable to get a nice performance boost for most users.

The reason I am asking is I am migrating a large code from Polymer to LitElement. Our main entry point does not use any polymer, so it would only be loaded once a user lands on some legacy pages. I'd also like to avoid loading `ShadyCSS` later on as well, but that might throw off some elements built with `LitElement` if `ShadyCSS` suddenly shows up after they have already rendered initially. I'd prefer it if `LitElement` did not deal with `ShadyCSS` at all on modern browsers.

**Comment by CaptainCodeman** [comment]
It's still needed to support polyfilled browers without native support for shadow DOM, it's not _just_ for `@apply`

**Comment by LarsDenBakker** [comment]
Of course, but as far as I can tell that is covered within the static styles already.

**Comment by jhpratt** [comment]
See also: #612.
